include/nccl_ofi_rdma.h

```diff
@@ -381,6 +381,15 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	 * response message */
 	nccl_ofi_rdma_connection_info_t conn_msg;
 
+	/*
+	 * Due to out-of-order delivery, it might happen that a
+	 * control message is received before the connect response
+	 * message. In this case we save the control message in
+	 * the send communicator until the connect response message
+	 * is received.
+	 */
+	nccl_ofi_deque_t *early_ctrl;
```

Comment:
I don't think we should do this; see https://github.com/aws/aws-ofi-nccl/pull/361#issuecomment-1998911238

Reply:
Yes, it's in its own commit, but given there are obviously two series of patches that run into this problem, this is the kind of issue that really needs to be hoisted into its own PR.  Please make it so.

Reply:
Sorry, I didn't understand this comment.

This PR had introduced some new `early_ctrl` behavior, and I was suggesting that we don't need that behavior and to remove it from this PR. I don't think we need a separate PR. Also not sure which other patch series you are referring to.